src/doc/trpl/comments.md

```diff
@@ -1,47 +1,45 @@
 % Comments
 
-Now that we have some functions, it's a good idea to learn about comments.
+Now that we have some functions, it’s a good idea to learn about comments.
```

Comment:
I really miss typographic quotes as well, but I think we ought to be consistent. We should run smartypants at build-time. Hoedown ships with a smartypants binary which we ought to add to the existing rust wrapper.